Mobile/iOS/Templates/Release_3.5.3/xxx_enforce_passcode.md

```diff
@@ -0,0 +1,33 @@
+###  Enforce passcode 
+
+#### Pr: https://github.com/owncloud/ios/pull/XXX 
+
+
+---
+
+ 
+| TestID | Test Case | Steps | Expected Result | Result | Related Comment |
+| :----: | :-------- | :---- | :-------------- | :----: | :-------------- |
+|**Install & Upgrade**||||||
+| 1 | Install from scratch | With the passcode enforced, install the app from scratch | Passcode is required, before the login view. Check the view in both orientations |  |
+| 2 | Upgrade without previous passcode | 1. Install a older version<br>2. Passcode disabled<br>3. Upgrade the app enforcing the passcode| Passcode is required |  |
+| 3 | Upgrade with previous passcode | 1. Install a older version<br>2. Passcode enabled<br>3. Upgrade the app enforcing the passcode| Passcode is not required |  |
```

Comment:
If passcode was enabled, expected: 1.Passcode is required 
2. Passcode can not be disabled in settings view